tests/slow/ParallelRestoreCorrectnessAtomicOpTinyData.txt

```diff
@@ -0,0 +1,62 @@
+testTitle=BackupAndParallelRestoreWithAtomicOp
+    testName=AtomicOps
+    nodeCount=30000
+;    transactionsPerSecond=2500.0
+;    transactionsPerSecond=500.0
+    transactionsPerSecond=100.0
+;    nodeCount=4
+;    transactionsPerSecond=250.0
+    testDuration=30.0
+    clearAfterTest=false
+
+; AtomicBackupCorrectness.txt does not mix Cycle and AtomicOps workloads
```

Comment:
I commented out those test scenarios instead of removing them on purpose.
I plan to increase the load of the tests and uncommenting those test scenarios later.